src/pages/index.js

```diff
@@ -5,54 +5,37 @@ import Card from "../components/Card.js";
 import Section from "../components/Section.js";
 import ModalWithForm from "../components/ModalWithForm.js";
 import ModalWithImage from "../components/ModalWithImage.js";
+import ModalWithConfirm from "../components/ModalWithConfirm.js";
+import Api from "../components/Api.js";
 import UserInfo from "../components/UserInfo.js";
 import "../pages/index.css";
 import {
   inputTitle,
   inputSubTitle,
+  inputPictureUrl,
   profileEditButton,
   formPerson,
   formImg,
+  changePfpForm,
   modalAdd,
+  profileSaveButton,
+  cardSaveButton,
+  pfpSaveButton,
 } from "../utils/constants.js";
+
 /*********************************************
-Initial cards array
+API on top to avoid scope issues
 **********************************************/
-
-const initialCards = [
-  {
-    name: "Yosemite Valley",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/yosemite.jpg",
-  },
-  {
-    name: "Lake Louise",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lake-louise.jpg",
-  },
-  {
-    name: "Bald Mountains",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/bald-mountains.jpg",
-  },
-  {
-    name: "Latemar",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/latemar.jpg",
-  },
-  {
-    name: "Vanoise National Park",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/vanoise.jpg",
-  },
-  {
-    name: "Lago di Braies",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lago.jpg",
-  },
-];
+const newApi = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+});
 
 /*********************************************
 Functions
 **********************************************/
-/*********************************************
-Sprint 8 start
-**********************************************/
-//make sure to define the functions before trying to call them.
+
```

Comment:
Needs correction: the avatar resets to default when I reload the page

Reply:
Hello, thanks for the review. I found that the name and job also reset on page refresh, but it seems like I am missing why this happens. I don't see any errors in the dev tools. Do you have any insight on this? I appreciate your help.